src/test/java/TestingEngine.java

```diff
@@ -0,0 +1,59 @@
+import java.util.Collections;
+import org.checkerframework.com.google.common.collect.Lists;
+import org.eclipse.jdt.core.dom.AST;
+import org.eclipse.jdt.core.dom.ASTParser;
+import org.eclipse.jdt.core.dom.CompilationUnit;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Utility class for other testing classes to use to run tests
+ */
+public class TestingEngine {
+	/**
+	 * RefactoringEngine to use to run tests
+	 */
+	private static RefactoringEngine fullEngine = new RefactoringEngine(
+			Lists.newArrayList("AddNullCheckBeforeDereferenceRefactoring"));
+
+	private static ASTParser parser = ASTParser.newParser(AST.getJLSLatest()); // Use appropriate
```

Comment:
Using the latest JLS makes me a bit nervous, since new JLS releases might cause us to break. It's probably okay, but we might want to write variants of this for each Java version that we want to support. Add a TODO here?